deposit-app/src/main/java/edu/unc/lib/boxc/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -898,8 +905,13 @@ private void addIngestionEventForChild(ContentObject obj) throws IOException {
         PremisEventBuilder builder = premisLogger.buildEvent(Premis.Ingestion);
 
         if (obj instanceof FileObject) {
-            builder.addEventDetail("ingested as PID: {0}\n ingested as filename: {1}",
-                    obj.getPid().getQualifiedId(), ((FileObject) obj).getOriginalFile().getFilename());
+            try {
+                var originalFile = ((FileObject) obj).getOriginalFile();
+                builder.addEventDetail("ingested as PID: {0}\n ingested as filename: {1}",
+                        obj.getPid().getQualifiedId(), originalFile.getFilename());
+            } catch (Exception e) {
+                builder.addEventDetail("ingested as PID: {0}", obj.getPid().getQualifiedId());
+            }
```

Comment:
"Exception" can cover a lot of stuff, what error was being thrown when there was no originalFile? I'd prefer switching this over to an `if` statement which detects when there is no originalFile, so we don't inadvertently end up skipping over unexpected error cases.

Reply:
I did have it as an if statement but the test kept failing because the ingest wouldn't work. it kept failing right after the original file is null check. so i decided to catch the error instead. it didn't tell me what kind of error so i just made it an exception. we could try debugging it some more?